### PR TITLE
dynamic judgment PASCAL

### PR DESCRIPTION
**Pull request by yajiedesign**

### CODE REVIEW

File: mshadow/dot_engine-inl.h [hunk 1]
```diff
@@ -429,28 +429,30 @@ struct BLASEngine<gpu, half::half_t> {
...
-    CHECK_EQ(err, CUBLAS_STATUS_SUCCESS) << "Cublas SgemmEx fail";
-#else
-    cublasStatus_t err = cublasSgemmEx(Stream<gpu>::GetBlasHandle(stream),
-                GetT(transa), GetT(transb), m, n, k, &alpha_f,
-                A, CUBLAS_DATA_HALF, lda, B, CUBLAS_DATA_HALF,
-                ldb, &beta_f, C, CUBLAS_DATA_HALF, ldc);
-    CHECK_EQ(err, CUBLAS_STATUS_SUCCESS) << "Cublas SgemmEx fail";
-#endif  // CUDA_VERSION >= 8000
-#endif  // MSHADOW_USE_PASCAL == 1
+	  if (stream->prop.major <= 5 && stream->prop.minor <= 2) {
```
Comment (piiswrong) [inline mshadow/dot_engine-inl.h]: indentation

**Comment by piiswrong** [comment]
please fix tests

**Comment by piiswrong** [comment]
its probably better to give it a default value, otherwise user code in mxnet could be broken.

**Comment by yajiedesign** [comment]
@piiswrong default values may hide some must be modified

**Pull request merged**